style.css

```diff
@@ -4,35 +4,62 @@
 
 body {
     box-sizing: border-box;
-    padding-top: 2rem;
+    padding-top: 7rem;
     display: none;
     margin: 0;
     font-family: 'Inter', sans-serif;
     text-align: center;
-    background-image: url("back-image.jpg");
+    /* background-image: url("back-image.jpg");
     background-repeat: no-repeat;
-    background-size: cover;
+    background-size: cover; */
 }
 
-nav {
+h1 {
+    margin: 0;
+}
+
+#background-icons>svg:nth-child(1) {
+    position: absolute;
+    right: 0;
+    top: 3.5rem;
+}
+
+#background-icons>svg:nth-child(3) {
+    position: absolute;
+    left: 0;
+    top: 31rem;
+}
+
+#background-icons>svg:nth-child(2) {
+    position: absolute;
+    right: 0;
+    top: 15rem;
+}
+
+header {
     box-sizing: border-box;
     margin: 0;
-    padding: 0.5rem 1rem;
+    padding: 0.5rem 1.5rem;
     display: flex;
     justify-content: space-between;
     position: absolute;
     top: 0;
     width: 100vw;
+    font-size: 0.8rem;
 }
 
-/* header {
-    margin-top: 3rem;
-    margin-bottom: 1rem;
-} */
-
-h1 {
+section h1 {
     margin: 0;
-    /* font-weight: 500; */
+    font-size: 2.5rem;
+}
```

Comment:
- [ ] Good job using the correct font size for this `h1`. Please consider giving this `h1` all the required style properties as seen in your design template. Add the properties that you don't have currently.

Figma design:
<img width="132" alt="Loom_P368UFVItO" src="https://user-images.githubusercontent.com/81468675/148405828-ff3b36d1-87f8-43a8-a5f9-08c80fd00957.png">

Reply:
![font h1](https://user-images.githubusercontent.com/83017311/148422373-e85b7c73-073c-4d4a-945c-fc9f57d98ca6.jpg)

`H1` already has those properties by default. Please check from `developer tools`.